japan/cndt-2022/OWNERS

```diff
@@ -0,0 +1,16 @@
+# See the OWNERS docs: https://go.k8s.io/community/contributors/guide/owners.md
+
+reviewers:
+  - oomichi
+  - atoato88
+  - shu-mutou
+  - s-kawamura-w664
+  - t-inu
+  - ziyi-xie
+approvers:
+  - oomichi
+  - atoato88
+  - shu-mutou
+  - s-kawamura-w664
+  - t-inu
+  - ziyi-xie
```

Comment:
Why do we need to add OWNERS file for each event?
I guess `japan/OWNERS` file could be enough for all events.
Or do we want to clarify who are instructors from each OWNERS file for each event?

Reply:
+1
I agree with your comment.
I'll remove OWNERS in this folder from PR.

Reply:
done.